src/main/resources/templates/fragments/fileUploader.html

```diff
@@ -431,7 +430,8 @@
       },
       error: function (file, errorMessage, xhr) {
         if (xhr && xhr.response) {
-          file.previewElement.getElementsByClassName("text--error")[0].innerText = xhr.response;
+          // redirect to client info page with intercepted param for error popup
+          location.href = "/flow/laDocUpload/clientInfo?intercepted=uploadDocuments"
```

Comment:
@sree-cfa I am not super pleased with this - as there is a possibility of other xhr related errors where we would have a response... but unfortunately the response does not contain any useful message or status code so it's hard to isolate

Reply:
In this case, I am seeing in the JS console a warning sending a request to the file upload endpoint as `XMLHttpRequest.readyState is not OPENED` when the browser cookies have been cleared. This error is not recoverable w/o resetting a session so I feel ok about having a client re-start - but I am not sure if there are others that would be recoverable